eng/scripts/Publish-SwiftMirrorRedirectReadme.ps1

```diff
@@ -0,0 +1,56 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $GitSourcePath,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $GitDestinationPath,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string] $UpstreamBranch = "master",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Repository = "azure"
+)
+
+$ErrorActionPreference = "Stop"
+
+. (Join-Path $PSScriptRoot/../common/scripts common.ps1)
+
+Push-Location $GitDestinationPath
+
+git checkout $UpstreamBranch
+
+$repoContents = (Get-ChildItem -Attributes Hidden,!Hidden -Exclude ".git").Name
+# If there is a diff from a single README.md file, clear the repo
+if (Compare-Object $repoContents @("README.md")) {
+    $repoContents | Remove-Item -Recurse -Force
+    LogDebug "Adding commit to clear $UpstreamBranch branch contents"
+    git add -u
+    git commit -m "Clear $UpstreamBranch branch contents"
+}
+
+Copy-Item -Path $GitSourcePath/eng/mirror.README.md -Destination README.md
+$repoUrl = git config --get remote.origin.url
+# Convert ssh remotes to https urls
+if ($repoUrl -imatch ".*($Repository/SwiftPM-.*)") {
+    $repoUrl = "https://github.com/$($matches[1])"
+}
+$readme = cat README.md
```

Comment:
Probably better to use Get-Content so we don't end up using different tools on different OS's.